src/components/Metrics/index.js

```diff
@@ -0,0 +1,100 @@
+import { RichTypography } from "@commons-ui/core";
+import { IconButton, Grid, useMediaQuery } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import DataVisualCard from "@/pesayetu/components/DataVisualCard";
+import Section from "@/pesayetu/components/Section";
+
+function Metrics({ items, sectionTitle, ...props }) {
+  const classes = useStyles(props);
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.only("md"));
+  if (!items?.length) {
+    return null;
+  }
+  return (
+    <div className={classes.root}>
+      <Section classes={{ root: classes.section }}>
+        <RichTypography variant="h2" className={classes.sectionTitle}>
+          {sectionTitle}
+        </RichTypography>
+        {items?.map((item, index) => (
+          <Grid
+            key={item.title}
+            container
+            direction={index === 0 ? "row" : "row-reverse"}
+            justifyContent="space-between"
+            alignItems="flex-start"
+            className={classes.metrics}
+          >
+            <Grid container item xs={12} md={5} lg={3}>
+              <Grid container item direction="row" alignItems="center">
+                <IconButton
+                  disableRipple
+                  disableFocusRipple
+                  color="primary"
+                  size="small"
+                  className={classes.button}
+                >
+                  <Image src={item.icon} width={44} height={44} />
+                </IconButton>
+                <RichTypography variant="h4" className={classes.title}>
+                  {item.title}
+                </RichTypography>
+              </Grid>
+              <RichTypography variant="body2" className={classes.description}>
+                {item.description}
+              </RichTypography>
+            </Grid>
+            <Grid
+              item
+              xs={12}
+              md={7}
+              lg={8}
+              container
+              direction="row"
+              justifyContent={
+                isTablet && index === 0 ? "flex-end" : "flex-start"
+              }
+              alignItems="center"
+            >
```

Comment:
There is only once component inside this Grid. Why do we need to flip `justifyContent`?

Reply:
@kilemensi This was to accomodate for the tablet device layout. Since we have to row reverse alternatively, the grid item containing the data visual coard will always have a flex-start. So added this to improve alignment